mirrord/cli/src/list.rs

```diff
@@ -219,7 +244,25 @@ pub(super) async fn print_targets(args: ListTargetArgs, rich_output: bool) -> Cl
         util::remove_proxy_env();
     }
 
-    let targets = FoundTargets::resolve(layer_config, rich_output).await?;
+    let target_types = if let Some(target_type) = args.target_type {
+        Some(vec![target_type])
+    } else {
+        match std::env::var(LS_TARGET_TYPES_ENV)
+            .ok()
+            .map(|val| serde_json::from_str::<Vec<Value>>(val.as_ref()))
```

Comment:
Why do we need to first convert it to `Value`? Can't we deserialize it into `Vec<TargetType>` here?